.ci/test-project/packages/magictestlib_cached/package.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
Can you change all these copyrights to the one at the top of `uberenv.py`.  These will never be upstreamed to Spack and probably shouldn't have their copyright banner on them.

https://github.com/LLNL/uberenv/blob/a4312b7d0ec578f0cb769d8d6357e4c9046f1512/uberenv.py#L3-L45